integration_tests/sdk/setup_integration.py

```diff
@@ -130,6 +130,14 @@ def _setup_snowflake_data(client: Client, snowflake: RelationalDBIntegration) ->
     _add_missing_artifacts(client, snowflake, existing_table_names)
 
 
+
+def _setup_relational_data(client: Client, relationalDB: RelationalDBIntegration) -> None:
```

Comment:
Copying over what @eunice-chan did in her SQLite PR:
https://github.com/aqueducthq/aqueduct/pull/942